controllers/operatorconfigmap_controller.go

```diff
@@ -205,13 +207,23 @@ func (c *OperatorConfigMapReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			return ctrl.Result{}, err
 		}
 
+		if err := c.storeSubscriptions(); err != nil {
```

Comment:
Why do we need a step for this?  it just creates a dependency between this being run and the reconcile functions that use the subscriptions. 

 I believe this is your attempt at a solution for the review comment from the last review, but it seems this creates more assumptions in the code. In my previous review, I alluded to Layzy loading the subscriptions in each reconcile function. That way the order does not matter, everything stays decoupled and it solves the issue of loading the subscription twice

Reply:
updated. done.